src/neo/IO/Caching/DataCache.cs

```diff
@@ -318,6 +318,12 @@ public TValue TryGet(TKey key)
             }
         }
 
+        public void AddAndCommit(TKey key, TValue value)
+        {
+            Add(key, value);
+            Commit();
+        }
```

Comment:
If this is used only in `Blockchain.cs`, then we needn't create a method in `DataCache`.